loader.qml

```diff
@@ -97,12 +233,20 @@ ApplicationWindow {
 
         Item {
             id: wallClock
-            property var time: new Date()
+            property var time: timewidget(settime.checked)
             Timer {
                 interval: 1000
                 running: true
                 repeat: true
-                onTriggered: wallClock.time = new Date()
+                onTriggered: wallClock.time = wallClock.timewidget(settime.checked)
+            }
+            function timewidget(statictime) {
```

Comment:
The norm is to use camelCase, so that would be timeWidget. Although it wouldn’t be clear what the function really does. I would expect timeWidget to be … a widget.
I would maybe rename it as getTimeToShow for example, then it’s apparent that it’s a function (get) that it returns a time, and that it has special logic to decide what needs to be shown. Also the function can just access settime.checked by itself, it is already tightly linked to other UI elements anyway

Reply:
Very helpful.  I named it `getDisplayTime()` and simplified as suggested